packages/cli/lib/lib/webpack/render-html-plugin.js

```diff
@@ -1,24 +1,50 @@
 const { resolve } = require('path');
-const { existsSync } = require('fs');
+const { existsSync, readFileSync, writeFileSync } = require('fs');
 const HtmlWebpackExcludeAssetsPlugin = require('html-webpack-exclude-assets-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 const prerender = require('./prerender');
 const createLoadManifest = require('./create-load-manifest');
 const { warn } = require('../../util');
 const { info } = require('../../util');
-let template = resolve(__dirname, '../../resources/template.html');
+let defaultTemplate = resolve(__dirname, '../../resources/template.html');
+
+function read(path) {
+	return readFileSync(resolve(__dirname, path), 'utf-8');
+}
 
 module.exports = async function(config) {
 	const { cwd, dest, isProd, src } = config;
 	const inProjectTemplatePath = resolve(src, 'template.html');
+	let template = defaultTemplate;
 	if (existsSync(inProjectTemplatePath)) {
 		template = inProjectTemplatePath;
 	}
+
+	template = config.template || template;
```

Comment:
Config template is `--template` option?
If yes I guess we should remove that and just stick to template.html in `src` because we scaffold that now

Reply:
I'm not sure I understand, it's the same logic as before. Do you want to remove the cli flag completely and hardcode `<project>/src/template.html`?

Reply:
Yeah like why to have two ways of doing the exact same thing? i.e. "custom template"
you want a custom template? just edit `template.html` is src. WDYT?

// @ForsakenHarmony

Reply:
but we can't have it break completely when someone deletes it in the project dir

Reply:
if they delete from there project i guess we start fallback to use the one in `preact-cli` repo

Reply:
https://github.com/preactjs/preact-cli/pull/862/files#diff-b26e9c1b1185752601692f2207489852R18-R21

Reply:
so we in theory have to check on every build / watch

Reply:
check on every build? yes.
check on **start** of every watch i guess.

Reply:
yeah

Reply:
So do we have a consensus of deleting `--template`?

Reply:
I think in either case that'd be something that should be addressed in a separate PR. This one is about changing the template itself and not the template resolution strategy.

If we settle on removing the `--template` argument we need to make sure to document it somehwere.